include/swift/AST/SearchPathOptions.h

```diff
@@ -398,6 +400,12 @@ class SearchPathOptions {
     Lookup.searchPathsDidChange();
   }
 
+  /// Retrieves the corresponding parent platform path for the SDK, or
+  /// \c nullopt if there isn't one.
+  /// NOTE: This computes and caches the result, and as such will not respect
+  /// a different FileSystem being passed later.
```

Comment:
Could we just set when setting the SDK path instead?

Reply:
It would need plumbing through in a whole bunch of places, e.g through argument parsing and deserialization, it didn't really seem worth it given I expect the FileSystem to always be `SourceMgr.getFileSystem()`